Frontend/ui-driver/src/Screens/HomeScreen/View.purs

```diff
@@ -105,6 +109,7 @@ screen initialState (GlobalState globalState) =
           _ <- pure $ spy "initial State" initialState
           _ <- HU.storeCallBackForNotification push Notification
           _ <- HU.storeCallBackTime push TimeUpdate
+          void $ launchAff $ flowRunner defaultGlobalState $ checkForAppUpdate push ShowAppUpdateBanner initialState 
```

Comment:
this function should not be called every time, add a condition flag to call this function for the first time while opening the app